.travis.yml

```diff
@@ -2,6 +2,7 @@ language: php
 
 php:
   - 7.1
+  - 7.2
```

Comment:
Shouldn't we leave it anyway?

Reply:
cc @joaorobertopb

Reply:
Humm ... In my opinion it would be interesting to have the tests run against php7.2 once it is stable.